attacks/locsearchadv/method.py

```diff
@@ -15,51 +15,6 @@ class LocSearchAdv(AttackMethod):
     LB = -1 #This is just a guess for what the lower and upper bounds should be
     UB = 1  #This is just a guess for what the lower and upper bounds should be
 
-    def cyclic(self, I, r, b, x, y): 
```

Comment:
Next time please don't change the code directly. Ask and let me and Alice do the changes as a way to show respect to others work!

Reply:
I switched the position of do_perturbation function to the front to take a screen shot. Sorry for not changing it back. The content of any functions you wrote remain intact.